modules/ROOT/pages/actions/routing.adoc

```diff
@@ -0,0 +1,20 @@
+:description: HTTP API Routing.
+
+[[http-api-actions-cluster-routing]]
+= Clusters and Routing
+
+The Cypher Transactional API provides a limited level of support for query routing.
+The level of support depends on whether xref:4.4-preview@operations-manual:ROOT:clustering/internals.adoc#causal-clustering-routing[Server-side Routing]  has been enabled.
```

Comment:
```suggestion
The level of support depends on whether link:https://neo4j.com/docs/operations-manual/current/clustering/internals/#clustering-routing[Server-side Routing] has been enabled.
```

Reply:
This is how we link elsewhere between the manuals. Also, using `4.4-preview` will probably not work once 4.4 is released.